data/data/agent/files/usr/local/bin/start-cluster-installation.sh.template

```diff
@@ -39,10 +39,12 @@ done
 
 echo "All ${total_required_nodes} hosts are ready."
 
-api_vip=$(curl -s -S "${BASE_URL}/clusters" | jq -r .[].api_vip)
-if [ "${api_vip}" == null ]; then
-    echo "Setting api vip"
-    curl -s -S -X PATCH "${BASE_URL}/clusters/${cluster_id}" -H "Content-Type: application/json" -d '{"api_vip": "{{.APIVIP}}"}'
+if [[ "$total_required_nodes" > "1" ]]; then
```

Comment:
Instead of conditioning on the number of nodes, we can condition on whether the api vip was set in the AgentClusterInstall.  The api vip won't be set for SNO clusters in the AgentClusterInstall. This was the change I was suggesting:
```suggestion
if [[ "${APIVIP}" != "" ]]; then
```

Reply:
Yes. I don't think we should limit it by nodes. Rather, by ACI VIPs. No vips, none platform